spec/spec_helper/temporary_repos.rb

```diff
@@ -19,18 +19,18 @@ def repo_make(name)
       path = repo_path(name)
       path.mkpath
       Dir.chdir(path) do
-        `git init`
+        Pod::Executable.capture_command!('git', %w(init))
         repo_make_readme_change(name, 'Added')
-        `git add .`
-        `git commit -m "Initialized."`
+        Pod::Executable.capture_command!('git', %w(add .))
+        Pod::Executable.capture_command!('git', %w(commit -nm Initialized.))
```

Comment:
also added option `-n` here.

Reply:
yes, `-n` skips verification, since we're making the initial commit in the repo, it normally spits out an error message